Compiler/src/profiling/tracy.jl

```diff
@@ -7,9 +7,9 @@ import ..@noinline, ..Cint, ..Vector, ..push!, ..unsafe_convert, ..esc,
 _strpointer(s::String) = ccall(:jl_string_ptr, Ptr{UInt8}, (Any,), s)
 
 mutable struct TracySrcLoc
-    zone_name::Ptr{UInt8}
-    function_name::Ptr{UInt8}
-    file::Ptr{UInt8}
+    @atomic zone_name::Ptr{UInt8}
```

Comment:
@atomic doesn't seem to be defined at this point so you will have to do the non macro builting

Reply:
Weird. It built just fine for me locally.

Reply:
It was `@atomic :symbol` that didn't work I think?

Reply:
I think it was because this is in a submode but earlier the `@atomc` wasn't imported.